pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -98,8 +98,10 @@ type ActualStateOfWorld interface {
 
 	// IsVolumeAttachedToNode returns true if the specified volume/node combo exists
 	// in the underlying store indicating the specified volume is attached to
-	// the specified node.
-	IsVolumeAttachedToNode(volumeName v1.UniqueVolumeName, nodeName types.NodeName) bool
+	// the specified node, and false if either the combo does not exist, or the
+	// attached state is marked as uncertain.
+	// TODO (verult)
+	IsVolumeAttachedToNode(volumeName v1.UniqueVolumeName, nodeName types.NodeName) AttachState
```

Comment:
There are 18  relevant change  in https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/attachdetach/cache/actual_state_of_world_test.go.   please modify them.  :)

Reply:
thanks, updated!